Workbooks/WorkbooksMetadata.json

```diff
@@ -1522,15 +1522,15 @@
         "provider": "Microsoft Sentinel Community"
       },
       {
-        "workbookKey": "LogSources&AnalyticRulesCoverageWorkbook",
+        "workbookKey": "LogSourcesAndAnalyticRulesCoverageWorkbook",
         "logoFileName": "",
         "description": "This workbook is intended to show how the different tables in a Log Analytics workspace are being used by the different Microsoft Sentinel features, like analytics, hunting queries, playbooks and queries in general.",
         "dataTypesDependencies": [ ],
         "dataConnectorsDependencies": [],
-        "previewImagesFileNames": [ "LogSources&AnalyticRulesCoverageBlack.png", "LogSources&AnalyticRulesCoverageWhite.png" ],
+        "previewImagesFileNames": [ "LogSourcesAndAnalyticRulesCoverageBlack.png", "LogSourcesAndAnalyticRulesCoverageWhite.png" ],
         "version": "1.1",
         "title": "Log Sources & Analytic Rules Coverage",
-        "templateRelativePath": "LogSources&AnalyticRulesCoverage.json",
+        "templateRelativePath": "LogSourcesAndAnalyticRulesCoverage.json",
```

Comment:
Hey @elforb, as there is change in value of template Relative path, should the workbook name be renamed?           [Azure-Sentinel](https://github.com/Azure/Azure-Sentinel)/[Workbooks](https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks)/LogSources&AnalyticRulesCoverage.json , please check and update, thanks.

Reply:
@v-sabiraj if you would check the name of the workbook file you'll see that I have already handled this name change.